settings.php

```diff
@@ -301,6 +301,16 @@
             saml2_settings::OPTION_TOLOWER_EXACT,
             $toloweroptions));
 
+    // Requested Attributes.
+    $settings->add(new admin_setting_configtextarea(
+        'auth_saml2/requestedattributes',
+        get_string('requestedattributes', 'auth_saml2'),
+        get_string('requestedattributes_help', 'auth_saml2', ['example' => "<pre>
+urn:mace:dir:attribute-def:eduPersonPrincipalName
+urn:mace:dir:attribute-def:mail *</pre>"]),
```

Comment:
Maybe a minor point. Would it be better to prepare the example using html_writer here, instead of coding it directly?

Even though it's 100% correct, in general less mistakes with closing tags, etc would happen if everyone used the helper methods.

Reply:
Maybe, I mostly bring it up in reviews when it matters from an escaping point of view but here it's just the tags and the the writer way would end up much longer and less readable, so ... meh?